apps/bare-expo/package.json

```
@@ -88,7 +88,7 @@
     "@react-native-community/viewpager": "4.1.6",
     "expo": "~39.0.1",
     "expo-dev-menu": "0.0.2",
-    "expo-development-client": "0.0.3",
+    "expo-development-client": "0.0.4",
```

```suggestion
    "expo-development-client": "0.0.3",
```
We don't need to change the package version now.